src/utils/hooks/useListTemplates.tsx

```diff
@@ -19,37 +19,25 @@ type TemplatesData = {
   templatesByCategory: TemplatesByCategory
 }
 
-const emptyTemplateData = {
+const emptyTemplateData: TemplatesData = {
   templates: [],
   templatesByCategory: [],
 }
 
 const useListTemplates = (templatePermissions: TemplatePermissions, isLoading: boolean) => {
-  const [templatesData, setTemplatesData] = useState<TemplatesData>(emptyTemplateData)
-  const [error, setError] = useState('')
-  const [loading, setLoading] = useState(true)
-
-  const { data, error: apolloError } = useGetTemplatesQuery({
+  const { data, loading, error } = useGetTemplatesQuery({
     skip: isLoading,
-    // fetchPolicy: 'network-only',
+    fetchPolicy: 'cache-and-network',
```

Comment:
Caching the templates list makes the menus load immediately on subsequent re-loads, which improves user experience a lot.